src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -606,6 +611,22 @@ func (c *Client) FindRelatedBooks(ctx context.Context, req *librarypb.FindRelate
     return it
 }
 
+// addLabel adds a label to the entity.
+func (c *Client) addLabel(ctx context.Context, req *taggerpb.AddLabelRequest) (*taggerpb.AddLabelResponse, error) {
+    md, _ := metadata.FromContext(ctx)
+    ctx = metadata.NewContext(ctx, metadata.Join(md, c.metadata))
+    var resp *taggerpb.AddLabelResponse
+    err := gax.Invoke(ctx, func(ctx context.Context) error {
+        var err error
+        resp, err = c.labelerClient.addLabel(ctx, req)
+        return err
+    }, c.CallOptions.AddLabel...)
```

Comment:
I guess we have to make the CallOptions field public even though the method is private. Fine for now. We may have to revisit someday.